package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "azure-managed-identity-config",
+  "version": "1.0.0",
+  "description": "",
+  "main": "ts-node run.ts",
+  "dependencies": {
+    "@pulumi/azure": "^0.17.4",
+    "@pulumi/pulumi": "^0.17.6",
```

Comment:
Did you end up using these anywhere?

Reply:
about to, working on it now, added that a bit early...